foia_hub/management/commands/load_agency_contacts.py

```diff
@@ -22,9 +15,33 @@ class Command(BaseCommand):
     """
 
     def handle(self, *args, **options):
-        try:
+        if len(args) > 0:
             yaml_folder = args[0]
-        except IndexError:
-            yaml_folder = _get_yaml_folder()
+        else:
+            yaml_folder = download_data()
 
+        print("Data directory: %s" % yaml_folder)
```

Comment:
@khandelwal please keep in mind that CF captures stdout and writes it to logs so it is always better to do stdout rather than logging to files that can be lost